README.md

```diff
@@ -187,75 +143,76 @@ At the root of this repository, execute `setup.sh`. It will:
-from the root of the directory.
-A successful output will contain:
+Validating these three deployments is done by executing:
+
+```console
+make validate
+```
+
+A successful output will look like this:
 ```console
 Validating Debian VM Webapp...
-Testing endpoint http://[ip-1]:8080
-Endpoint http://[ip-1]:8080 is responding.
+Testing endpoint http://35.227.149.80:8080
+Endpoint http://35.227.149.80:8080 is responding.
+**** http://35.227.149.80:8080/prime/10
 The sum of all primes less than 10 is 17
 The factorial of 10 is 3628800
 
 Validating Container OS Webapp...
-Testing endpoint http://[ip-2]:8080
-Endpoint http://[ip-2]:8080 is responding.
+Testing endpoint http://35.230.123.231:8080
+Endpoint http://35.230.123.231:8080 is responding.
+**** http://35.230.123.231:8080/prime/10
 The sum of all primes less than 10 is 17
 The factorial of 10 is 3628800
 
 Validating Kubernetes Webapp...
-Testing endpoint http://[ip-3]
-Endpoint http://[ip-3] is responding.
+Testing endpoint http://35.190.89.136
+Endpoint http://35.190.89.136 is responding.
+**** http://35.190.89.136/prime/10
 The sum of all primes less than 10 is 17
 The factorial of 10 is 3628800
 ```
 
+Of course, the IP addresses will likely differ for your deployment.
+
 ## Load Testing
 
-In a new console window, execute (using ip-3 from above)
+In a new console window, execute the following, replacing `[IP_ADDRESS]` with the IP address and port from your validation output from the previous step. Note that the Kubernetes deployment runs on port `80`, while the other two deployments run on port `8080`:
 ```console
-ab -c 120 -t 60  http://[ip-3]/prime/10000
+ab -c 120 -t 60  http://[IP_ADDRESS]/prime/10000
 ```
-ApacheBench (`ab`) will execute 120 concurrent requests against ip-3 for 1
-minute. This single replica is insufficient to handle this volume of requests.
+ApacheBench (`ab`) will execute 120 concurrent requests against the provided endpoint for 1 minute. The demo application's replica is insufficiently sized to handle this volume of requests.
 
-![screenshot](./images/ab_load-test-1.png)
+This can be confirmed by reviewing the output from the `ab` command. A `Failed requests` value of greater than 0 means that the server couldn't respond successfully to this load:
 
-This can be confirmed by reviewing the output from the `ab` command. A
-"Failed requests" value of more than 0 means that the server couldn't
-respond successfully for this amount of load.
+![screenshot](./images/ab_load-test-1.png)
 
-A good mechanism for scaling this type of service is via horizontal scaling
-(adding additional instances behind a load balancer). For either of the
-first two deployments (Debian or COS), this would take several steps:
+A good mechanism for scaling this type of service is via horizontal scaling (adding additional instances behind a load balancer). For either of the first two deployments (Debian or COS), this would take several steps:
```

Comment:
One way to ensure that your system has capacity to handle this type of traffic is by scaling up. In this case, we would want to [scale our service horizontally](https://en.wikipedia.org/wiki/Scalability#Horizontal_and_vertical_scaling).

In our Debian and COS architectures, horizontal scaling would include: 
1. Creating a load balancer
1. Spinning up additional instances
1. Registering them with the load balancer

Reply:
👍